exporters/otlp/internal/transform/attribute_test.go

```diff
@@ -146,3 +148,147 @@ func TestAttributes(t *testing.T) {
 		}
 	}
 }
+
+func TestArrayAttributes(t *testing.T) {
```

Comment:
I create a separate method for array tests, as there are multiple test cases and it would clutter the previous one.